src/main.rs

```diff
@@ -95,15 +95,15 @@ fn main() {
     let export_manager = build_export_manager(&matches);
     let commands = build_commands(&matches);
 
-    let res = match options {
-        Ok(opts) => run(&commands, &opts),
-        Err(e) => error(e.description()),
+    let res = match &options {
+        &Ok(ref opts) => run(&commands, &opts),
```

Comment:
`ref` is ok here, but why match on a borrowed `&options` instead of just `options`?

Reply:
huh...I got here via a circuitous route and looks like `ref` fixed the issue that I first tried solving with borrowing.

Fixed :)